src/mscorlib/src/System/Reflection/Emit/DynamicILGenerator.cs

```diff
@@ -241,6 +241,44 @@ public override void Emit(OpCode opcode, String str)
             PutInteger4(token);
         }
 
+        public override void EmitCalli(OpCode opcode, CallingConvention unmanagedCallConv, Type returnType, Type[] parameterTypes)
+        {
+            int stackchange = 0;
+            int cParams = 0;
+            int i;
+            SignatureHelper sig;
+
+            // The opcode passed in must be the calli instruction.
+            Debug.Assert(opcode.Equals(OpCodes.Calli),
```

Comment:
We do not do argument validation of public APIs using asserts. It is unfortunate that this method takes ignored useless argument, but it should be just ignored (it was ignored in .NET Framework).